rooms.js

```diff
@@ -163,6 +165,107 @@ var Room = (function () {
 		}
 		return true;
 	};
+	//mute handling
+	Room.prototype.runMuteTimer = function () {
+		if (this.muteTimer || this.mutedUsers.length === 0) return;
```

Comment:
What happens if you mute someone for 60 minutes, then mute someone else for 7 minutes? It seems like `this.muteTimer` already exists when the 7-min mute happens, so the timer isn't updated.